source/Government.h

```diff
@@ -20,6 +20,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "ExclusiveItem.h"
 #include "LocationFilter.h"
 
+#include <functional>
```

Comment:
unneeded include

Reply:
its for std::pair

Reply:
`std::pair` comes from the header `<utility>`.
`<functional>`: https://en.cppreference.com/w/cpp/header/functional